tools/templates/ci.yml

```diff
@@ -50,12 +50,34 @@ jobs:
       command: build
       projects: '$(file)'
       arguments: '--configuration Release'
+  - task: DotNetCoreCLI@2
+    inputs:
+      command: 'publish'
+      arguments: '--configuration Release --output $(Build.ArtifactStagingDirectory)/$(agent)_Release'
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 'Software BOM'
+    inputs:
+      BuildDropPath: '$(System.ArtifactsDirectory)/$(agent)_Release'
+      PackageName: '$(agent)_Release'
+  - publish: $(System.ArtifactsDirectory)/$(agent)_Release
+    artifact: $(agent)_Release
   - task: DotNetCoreCLI@2
     displayName: Debug Build
     inputs:
       command: build
       projects: '$(file)'
       arguments: '--configuration Debug'
+  - task: DotNetCoreCLI@2
+    inputs:
```

Comment:
Please add a descriptive `displayName` to the task. It shows up as `DotNetCoreCLI` now.